src/notename/johnston.rs

```diff
@@ -129,25 +129,33 @@ pub mod fivelimit {
 
             let sf = self.accidental.sharpflat;
             if sf > 0 {
-                for _ in 0..sf {
-                    write!(f, "#")?;
+                if sf == 2 {
+                    write!(f, "\u{1D12A}")?; // double sharp
+                } else {
+                    for _ in 0..sf {
+                        write!(f, "\u{266F}")?; // sharp
+                    }
                 }
             }
             if sf < 0 {
-                for _ in 0..-sf {
-                    write!(f, "b")?;
+                if sf == -2 {
+                    write!(f, "\u{1D12B}")?; // double flat
+                } else {
+                    for _ in 0..-sf {
+                        write!(f, "\u{266D}")?; // flat
+                    }
```

Comment:
Wow, that's nice to have double flats and sharps! But this code handles notes with more than three accidentals incorrectly:
<img width="283" height="305" alt="20251102_15h16m56s_grim" src="https://github.com/user-attachments/assets/5c9eccfc-c331-4cf3-8284-c4b7f3b55176" />
I think you should write something like
```rust
for _ in 0..(sf / 2) {
  write!(f, "\u{1D12B}")?; // double flat
}
if sf % 2 == 1 {
  write!(f, "\u{266D}")?; // flat
}
```
That is: write as many double flats (sharps) as possible, and then at most one flat (sharp).